pubsub/topics/publish_settings.go

```diff
@@ -35,9 +33,8 @@ func publishWithSettings(w io.Writer, projectID, topicID string) error {
 	if err != nil {
 		return fmt.Errorf("pubsub.NewClient: %v", err)
 	}
-
-	var wg sync.WaitGroup
-	var totalErrors uint64
+	var resultSlice []*pubsub.PublishResult
```

Comment:
```suggestion
	var results []*pubsub.PublishResult
```

Generally, you don't need to include "Slice" in the name for slices.

Reply:
Ok, done